rosco-web/src/main/groovy/com/netflix/spinnaker/rosco/controllers/BakeryController.groovy

```diff
@@ -78,12 +78,18 @@ class BakeryController {
     return baseImage
   }
 
-  @ExceptionHandler
+  @ExceptionHandler(BakeOptions.Exception)
   @ResponseStatus(HttpStatus.NOT_FOUND)
   Map handleBakeOptionsException(BakeOptions.Exception e) {
     [error: "bake.options.not.found", status: HttpStatus.NOT_FOUND, messages: ["Bake options not found. " + e.message]]
   }
 
+  @ExceptionHandler(Exception)
```

Comment:
Please file an issue on https://github.com/spinnaker/spinnaker/issues with some steps to reproduce the null message text and we'll fix it as a separate pr. Will need to test any fix with the full spinnaker stack, so it'll be easier to tackle it independently.